src/infra/infra.go

```diff
@@ -41,7 +41,7 @@ func CreateInfra(configuration exargs.Configuration) (Infra, error) {
 	for name, path := range configuration.Rooms {
 		b, err := GetBricks(name, path)
 		if err != nil {
-			fmt.Printf(`Cannot add "%s" (path: %s) room's bricks: %s`, name, path, err)
+			fmt.Printf("Cannot add \"%s\" (path: %s) room's bricks: %s\n", name, path, err)
```

Comment:
fmt.Printf("Cannot add \"%s\" (path: %s) room's bricks: %v\n", name, path, err)

The last %s should be %v

Reply:
I looked it up 'cause I wasn't sure it's that clear cut, especially considering it works. But I think it's up for debate: https://stackoverflow.com/questions/43887776/what-is-the-correct-format-specifier-to-print-error-object-in-go-s-or-v

However, considering that's how we do it in most of the other code, I'm fine with the change, sure :ok_hand: